packages/react-core/src/components/ToggleGroup/ToggleGroup.tsx

```diff
@@ -9,6 +10,8 @@ export interface ToggleGroupProps extends React.HTMLProps<HTMLDivElement> {
   className?: string;
   /** Modifies the toggle group to include compact styling. */
   isCompact?: boolean;
+  /** Disable all ToggleGroup items under this component. */
```

Comment:
One last nit, is there a reason you kept `ToggleGroup` rather than adding the space and changing capitalization to `toggle group`? My understanding of [the capitalization guidelines](https://www.patternfly.org/v4/ux-writing/capitalization/#capitalization-types) is that components named in text that gets exposed in the docs should only be capitalized when starting a sentence.

Reply:
Thanks for referencing me to the guidelines. I didn't know it mentions this. I will update this line.
Appreciate the feedback.